cognite/client/data_classes/datapoints.py

```diff
@@ -847,7 +847,7 @@ def __str__(self) -> str:
         item = self.dump()
         for i in item:
             i["datapoints"] = convert_and_isoformat_time_attrs(i["datapoints"])
-        return json.dumps(item, default=lambda x: x.__dict__, indent=4)
+        return _json.dumps(item, default=lambda x: x.__dict__, indent=4)
```

Comment:
I think we can remove this: `default=lambda x: x.__dict__`

Reply:
Nice. Will try that!